### PR TITLE
Fixing the create schema functionality

### PR DESCRIPTION
**Pull request by jlew25**
Create schema needs more complexity in regards to connecting to Redshift so we are adjusting it as such.

### CODE REVIEW

File: s3parq/publish_parq.py [hunk 1]
```diff
@@ -170,6 +184,7 @@ def publish(bucket: str, key: str, partitions: iter, dataframe: pd.DataFrame, re
         ARGS: 
             Schema_name: str
             Table_name: str
+            iam_role
```
Comment (schafrn) [inline s3parq/publish_parq.py]: Please add the :str here and lowercase schema_name and table_name. It's so inconsequential but. It's needed.

File: s3parq/publish_parq.py [hunk 2]
```diff
@@ -206,7 +222,7 @@ def publish(bucket: str, key: str, partitions: iter, dataframe: pd.DataFrame, re
             db_name = redshift_params['db_name']
         )
         session_helper.configure_session_helper()
-        create_schema(redshift_params['schema_name'], session_helper)
+        create_schema(redshift_params['schema_name'], redshift_params['db_name'], redshift_params['iam_role'], session_helper)
```
Comment (schafrn) [inline s3parq/publish_parq.py:225]: Can you use the iam role off of the session helper here? Or is that unintended?

Comment (schafrn) [reply s3parq/publish_parq.py:225]: Misread that nvm

File: s3parq/publish_parq.py (hunk 1 above)
Comment (jlew25) [reply s3parq/publish_parq.py]: fixed

**Review by natie-ichain — APPROVED** [review]
great work justin :)

**Pull request merged**